src/Infrastructure.EntityFramework/Repositories/DatabaseContext.cs

```diff
@@ -141,7 +135,9 @@ protected override void OnModelCreating(ModelBuilder builder)
             eUser.ToTable(nameof(User));
             eOrganizationApiKey.ToTable(nameof(OrganizationApiKey));
             eOrganizationConnection.ToTable(nameof(OrganizationConnection));
-            eSecret.ToTable(nameof(Secret));
+
+            // Apply all configurations specified in types implementing IEntityTypeConfiguration in the assembly.
+            builder.ApplyConfigurationsFromAssembly(typeof(SecretEntityTypeConfiguration).Assembly);
```

Comment:
I assume this is to get the assembly of the Infrastructure.EF project? Should we use the `DatabaseContext` file instead which should have the same effect but is less coupled to a single configuration.

Reply:
Resolved in https://github.com/bitwarden/server/pull/2144/commits/9c410672d8b381974f1ab20a384028eb21fab8ed